composer.json

```diff
@@ -32,5 +32,10 @@
         "psr-4": {
             "LicenseChecker\\Tests\\": "tests/"
         }
+    },
+    "config": {
+        "allow-plugins": {
```

Comment:
I added this property because it will be required by Composer in the very near future. See https://getcomposer.org/doc/06-config.md#allow-plugins